packages/console/src/components/PlanUsage/PlanUsageCard/index.tsx

```diff
@@ -34,42 +35,57 @@ function ProPlanUsageCard({
 }: Props) {
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
 
+  const usagePercent = conditional(
+    typeof quota === 'number' && typeof usage === 'number' && usage / quota
+  );
+
   return (
     <div className={classNames(styles.card, className)}>
       <div className={styles.title}>
         <span>
           <DynamicT forKey={titleKey} />
         </span>
-        <ToggleTip
-          content={
-            <Trans
-              components={{
-                a: <TextLink to="https://blog.logto.io/pricing-add-ons/" />,
-              }}
-            >
-              {t(tooltipKey, {
-                price: unitPrice,
-              })}
-            </Trans>
-          }
-        >
-          <IconButton size="small">
-            <Tip />
-          </IconButton>
-        </ToggleTip>
+        {tooltipKey && (
+          <ToggleTip
+            content={
+              <Trans
+                components={{
+                  a: <TextLink to="https://blog.logto.io/pricing-add-ons/" />,
```

Comment:
Better put this link in the `consts/external-links.ts`, as I saw it numerous times

Reply:
See LOG-9709, we can update this when we change the blog link.